app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -616,6 +616,7 @@
   @event={{this.data.event}}
   @loading={{this.loading}}
   @first={{true}}
+  @device={{this.device}}
```

Comment:
what about other places.
there are 4 places where we are using this object.
update on other three places

Reply:
done.